### PR TITLE
Split "magical silver weapons or magical weapons shedding bright light" into two separate exceptions

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/shadows-at-sundown-bestiary/aliriel.json [hunk 1]
```diff
@@ -4424,18 +4424,21 @@
                 "rules": [
                     {
                         "exceptions": [
+                            {
+                                "definition": [
+                                    "item:magical",
+                                    "item:material:silver",
+                                    "item:type:weapon"
```
Comment (stwlam) [inline packs/shadows-at-sundown-bestiary/aliriel.json:4431]: Need to exclude unarmed attacks

File: packs/shadows-at-sundown-bestiary/aliriel.json [hunk 2]
```diff
@@ -4424,18 +4424,21 @@
...
+                                ],
```
Comment (stwlam) [inline packs/shadows-at-sundown-bestiary/aliriel.json:4432]:
```suggestion
                                "definition": [
                                    "item:magical",
                                    "item:material:silver",
                                    {
                                        "or": [
                                            "item:type:melee",
                                            "item:type:weapon"
                                        ]
                                    },
                                    {
                                        "not": "item:category:unarmed"
                                    }
                                ],
```

File: packs/shadows-at-sundown-bestiary/aliriel.json [hunk 3]
```diff
@@ -4427,15 +4427,26 @@
...
                                     }
                                 ],
-                                "label": "PF2E.IWR.Custom.MagicalSilverWeaponsOrMagicalWeaponsBrightLight"
+                                "label": "PF2E.IWR.Custom.MagicalSilverWeapons"
+                            },
+                            {
+                                "definition": [
+                                    "item:magical",
+                                    "item:type:weapon",
+                                    "item:weapons-shedding-bright-light"
```
Comment (simonward) [inline packs/shadows-at-sundown-bestiary/aliriel.json:4447]: Should we exclude unarmed here as well, given that it's not currently automated anyway?

Comment (simonward) [reply packs/shadows-at-sundown-bestiary/aliriel.json]: Yeah, probably.

**Pull request merged**